config/drupal_modules_disable.sh

```diff
@@ -7,7 +7,6 @@
 
 MODULES_DISABLE=(
   "color"
-  "overlay"
```

Comment:
I really don't like this module as well, but the functionality was already implemented before using the `overlay` module, and besides the jQuery version that caused some other small issues we didn't need to do much for this one.
I don't know what exactly was the reason to disable this module but it caused a big issue for this feature.

We'll check the mobile, although i'm not sure that drag n drop is available over there ;)